unison-src/transcripts-using-base/serial-test-00.output.md

```diff
@@ -80,7 +80,7 @@ mkTestCase = do
                    -> (Tree Text -> Text)
                    -> (Tree Nat, Tree Nat, Tree Nat, Tree Text)
                    -> Text
-      foldMap    : ∀ a r g2 g1 g.
+      foldMap    : ∀ r g2 g1 a g.
                      r
                      -> (r ->{g2} r ->{g1} r)
                      -> (a ->{g} r)
```

Comment:
Examples here showing the forall order normalization. Note that the order doesn't matter most of the time and won't even be shown unless an explicit `forall` is needed during pretty-printing.